utils/appliance.py

```diff
@@ -636,6 +647,51 @@ def precompile_assets(self, log_callback=None):
 
         return status
 
+    def backup_database(self, log_callback=None):
+        """Backup VMDB database
+
+        TO DO:
+            Remove hard coded NFS server name and get NFS server details from yaml file.
+
+        """
+        if log_callback is None:
+            log_callback = logger.info
+
+        log_callback('Backing up database')
+
+        client = self.ssh_client()
+        status, output = client.run_rake_command(
+            'evm:db:backup:remote -- --uri nfs://10.16.7.91/home/jenkins --remote-file-name backup')
```

Comment:
@mfalesni Yes,that's on my to do list :).

I'll take a look at your other comments.